draft-ietf-quic-multipath.md

```diff
@@ -382,55 +382,51 @@ Reset ({{Section 10.3 of QUIC-TRANSPORT}}) closes the connection.
 
 Both endpoints, namely the client and the server, can close a path,
 by sending PATH_ABANDON frame (see {{path-abandon-frame}}) which
-abandons the path with a corresponding Path Identifier. Once a path is
-marked as "abandoned", it means that the resources related to the path,
-such as the used connection IDs, can be released.
-However, information related to data delivered over that path SHOULD
-not be released immediately as acknowledgments can still be received
-or other frames that also may trigger retransmission of data on another
-path.
+abandons the path with a corresponding Path Identifier.
 
-The endpoint sending the PATH_ABANDON frame SHOULD consider a path as
+The endpoint sending the PATH_ABANDON frame can consider a path as
 abandoned when the packet that contained the PATH_ABANDON frame is
-acknowledged. When releasing resources of a path, the endpoint SHOULD
-send a RETIRE_CONNECTION_ID frame for the connection IDs used on the path,
-if any.
+acknowledged.
```

Comment:
not sure we need this sentence at all...

Reply:
Agree.

Reply:
```suggestion
```